zone/client.cpp

```diff
@@ -2698,6 +2698,38 @@ uint16 Client::GetMaxSkillAfterSpecializationRules(EQ::skills::SkillType skillid
 	return Result;
 }
 
+uint8 Client::GetPVPTeam() {
+	if (RuleI(World, PVPSettings) == 2) {
+		int myrace = GetRace();
+		// humans
```

Comment:
Instead of leaving comments here (which are still great and better than nothing)

It helps readability to define intent via boolean above and use it in your checks

```cpp
bool is_human = (myrace == 1 || myrace == 2 || myrace == 3 || myrace == 522);
```

The returns should be a constant somewhere instead of magic numbers, at the very least thrown in `eq_constants.h`